internal/hcs/process.go

```diff
@@ -448,6 +442,12 @@ func (process *Process) CloseStdin(ctx context.Context) (err error) {
 		}
 	}
 
+	process.stdioLock.Lock()
+	defer process.stdioLock.Unlock()
+	if process.stdin == nil {
+		return nil
+	}
+
 	process.stdin.Close()
 	process.stdin = nil
 
```

Comment:
A nit pick, but I'd suggest flipping the contents of the if statement so it matches the other two functions.

```suggestion
	if process.stdin != nil {
		process.stdin.Close()
		process.stdin = nil
	}
```

Reply:
Done! Thanks